Add Mezmo and Busybox files (#208)
odidev/ecosystem-dashboard-for-arm: content/opensource_packages/busybox.md

---
name: Busybox

> category: Miscellaneous

Author: check if category can be compilers/tools.

description: BusyBox is a collection of Unix utilities bundled together into one executable file.
download_url: https://www.busybox.net/downloads/binaries/
works_on_arm: true
supported_minimum_version:
    version_number: 1.28.1
    release_date: 15/02/2018

optional_info: 
    homepage_url: https://www.busybox.net/
    support_caveats:
    alternative_options:
    getting_started_resources:
        arm_content: https://learn.arm.com/learning-paths/servers-and-cloud-computing/refinfra-quick-start/build-2/
        partner_content:
        official_docs: https://www.busybox.net/FAQ.html#getting_started
    arm_recommended_minimum_version:
        version_number:
        release_date:

optional_hidden_info:
    release_notes__supported_minimum:
    release_notes__recommended_minimum:

>     other_info: Linux/ARM64 release notes are not available. The first Linux/armv8 tar is available in version v1.28.1, as can be found [here](https://www.busybox.net/downloads/binaries/1.28.1-defconfig-multiarch/).

Author: ```suggestion
    other_info: Linux/ARM64 release notes are not available. The first Linux/armv8 tar is available from version v1.28.1 and  can be found [here](https://www.busybox.net/downloads/binaries/1.28.1-defconfig-multiarch/).
```
